mmcv/ops/carafe.py

```diff
@@ -263,7 +263,7 @@ def init_weights(self):
     def kernel_normalizer(self, mask):
         mask = F.pixel_shuffle(mask, self.scale_factor)
         n, mask_c, h, w = mask.size()
-        mask_channel = int(mask_c / (self.up_kernel * self.up_kernel))
+        mask_channel = int(mask_c / float(self.up_kernel**2))
```

Comment:
Is the change here necessary? If so, please add a comment in the code to indicate the reason.